hazelcast/src/main/java/com/hazelcast/spi/impl/proxyservice/impl/ProxyServiceImpl.java

```diff
@@ -239,7 +239,7 @@ public void dispatchEvent(final DistributedObjectEventPacket eventPacket, Object
             try {
                 final ProxyRegistry registry = getOrCreateRegistry(serviceName);
                 if (!registry.contains(eventPacket.getName())) {
-                    registry.createProxy(eventPacket.getName(), false, true);
+                    registry.createProxy(eventPacket.getName(), false, true, true);
```

Comment:
Minor: Later we should change signature of this function, having many boolean arguments is confusing.
https://dave.cheney.net/2019/09/24/be-wary-of-functions-which-take-several-parameters-of-the-same-type